Ownership/Non-UK/Reykjavik.txt

```diff
@@ -1,7 +1,11 @@
-SECTOR:BIRD Low:0:24500
-OWNER:IR:RKE:RK
+SECTOR:BIRD Low:0:5500
+OWNER:RKL:RKH:RKE:RK
 BORDER:BIRD:Heb_BIRD Del Inner:Heb_BIRD:Mor_BIRD
 
-SECTOR:BIRD:24500:66000
-OWNER:IR:RKE:RK:EI
+SECTOR:BIRD Mid:5500:34500
+OWNER:RKL:RKH:RKE:RK:IR
```

Comment:
Assuming these ownerships can be changed by reducing the number of identifiers defined for the same frequency?